packages/gluestick/src/renderer/helpers/cacheManager.js

```diff
@@ -17,6 +24,13 @@ const getCacheKey = ({ hostname, url }: { hostname: string, url: string}): strin
 };
 
 module.exports = (logger: Logger, isProduction: boolean): CacheManager => {
+  const enableComponentCaching: EnableComponentCaching = (config) => {
+    if (isProduction && !!config && !!config.components) {
```

Comment:
Are those `!!` really needed?? Both `config` and `config.components` will be evaluated to boolean.